### PR TITLE
 update examples region to match the description in docs

### PR DESCRIPTION
**Pull request by sreevani-ship-it**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Example snippet was not matching the text description in the doc.
Issue Number: 28147

## What is the new behavior?

Updated example region to match the text description.
## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Comment by brandonroberts** [comment]
@chembu will you rebase to resolve the conflicts?

File: aio/content/examples/router/src/app/heroes/hero-list/hero-list.component.1.html [hunk 1]
```diff
@@ -2,8 +2,10 @@
 <h2>HEROES</h2>
 <ul class="items">
   <li *ngFor="let hero of heroes$ | async">
-    <!-- #docregion nav-to-detail -->
+    <!-- #docregion nav-to-detail-->
```
Comment (brandonroberts) [inline aio/content/examples/router/src/app/heroes/hero-list/hero-list.component.1.html]:
```suggestion
    <!-- #docregion nav-to-detail -->
```

File: aio/content/examples/router/src/app/heroes/hero-list/hero-list.component.1.html [hunk 2]
```diff
@@ -2,8 +2,10 @@
...
+    <!-- #docregion link-parameters-array-->
```
Comment (brandonroberts) [inline aio/content/examples/router/src/app/heroes/hero-list/hero-list.component.1.html]:
```suggestion
    <!-- #docregion link-parameters-array -->
```

File: aio/content/examples/router/src/app/heroes/hero-list/hero-list.component.1.html [hunk 3]
```diff
@@ -2,8 +2,10 @@
...
     <a [routerLink]="['/hero', hero.id]">
+      <!-- #enddocregion link-parameters-array-->
```
Comment (brandonroberts) [inline aio/content/examples/router/src/app/heroes/hero-list/hero-list.component.1.html]:
```suggestion
      <!-- #enddocregion link-parameters-array -->
```

**Review by brandonroberts — CHANGES_REQUESTED** [review]

File: aio/content/examples/router/src/app/heroes/hero-list/hero-list.component.1.html [hunk 4]
```diff
@@ -3,7 +3,9 @@ <h2>HEROES</h2>
 <ul class="items">
   <li *ngFor="let hero of heroes$ | async">
     <!-- #docregion nav-to-detail -->
+    <!-- #docregion link-parameters-array -->
     <a [routerLink]="['/hero', hero.id]">
+      <!-- #enddocregion link-parameters-array -->
```
Comment (brandonroberts) [inline aio/content/examples/router/src/app/heroes/hero-list/hero-list.component.1.html]:
```suggestion
    <!-- #enddocregion link-parameters-array -->
```

Comment (brandonroberts) [reply aio/content/examples/router/src/app/heroes/hero-list/hero-list.component.1.html]: The region matchers must line up

**Review by brandonroberts — CHANGES_REQUESTED** [review]

File: aio/content/guide/router.md [hunk 5]
```diff
@@ -2029,7 +2029,7 @@ When navigating to the `HeroDetailComponent` you specified the _required_ `id` o
 *route parameter* and made it the second item of the [_link parameters array_](#link-parameters-array).
 
 
-<code-example path="router/src/app/heroes/hero-list/hero-list.component.1.ts" linenums="false" header="src/app/heroes/hero-list/hero-list.component.ts (link-parameters-array)" region="link-parameters-array">
+<code-example path="router/src/app/heroes/hero-list/hero-list.component.html" linenums="false" header="src/app/heroes/hero-list/hero-list.component.html (link-parameters-array)" region="link-parameters-array">
```
Comment (brandonroberts) [inline aio/content/guide/router.md]:
```suggestion
<code-example path="router/src/app/heroes/hero-list/hero-list.component.1.html" linenums="false" header="src/app/heroes/hero-list/hero-list.component.html (link-parameters-array)" region="link-parameters-array">
```

**Review by brandonroberts — CHANGES_REQUESTED** [review]
Update example path

**Review by brandonroberts — APPROVED** [review]

**Review by jasonaden — CHANGES_REQUESTED** [review]
This needs a rebase.

Also, can't we remove the `link-parameters-array` section in [`hero-list.component.1.ts`](https://github.com/angular/angular/blob/master/aio/content/examples/router/src/app/heroes/hero-list/hero-list.component.1.ts#L33)?

**Comment by jasonaden** [comment]
@chembu This is still waiting for the rebase and the change above. Changing to `cleanup` status.

**Review by jasonaden — APPROVED** [review]

**Pull request closed without merging**